src/options.rs

```diff
@@ -14,27 +14,31 @@ impl Default for SetterPattern {
     }
 }
 
-#[derive(Debug)]
+#[derive(Debug, Clone)]
 pub struct Options {
     /// e.g. `#[setters]` (defaults to true)
     setter_enabled: bool,
     /// e.g. `#[setters(owned)]` (defaults to mutable)
     setter_pattern: SetterPattern,
     /// e.g. `#[setters(prefix="with")]` (defaults to None)
-    setter_prefix: Option<String>,
+    setter_prefix: String,
     /// e.g. `#[setters(private)]` (defaults to public)
     setter_public: bool,
     /// e.g. `#[setters(options="implicit")]` (defaults to explicit)
     setter_implicit_options: bool,
     /// e.g. `#[getters]` (defaults to false)
     getter_enabled: bool,
     /// e.g. `#[getters(prefix="with")]` (defaults to None)
```

Comment:
Oh right, users will need additional docs. I prefer doc comments for API-level comments, and regular double-slash comments for implementation details (that don't leak to the API level).

Keep the automatic underscore injection. It's what I'd expect.